code/game/machinery/Sleeper.dm

```diff
@@ -20,14 +20,17 @@
 	var/controls_inside = FALSE
 	var/max_vials = 6
 	var/list/inserted_vials = list()
+	var/list/roundstart_vials = list(
+		/obj/item/reagent_containers/chem_bag/morphine
+	)
 	var/list/roundstart_chems = list(
 		/datum/reagent/medicine/epinephrine,
-		/datum/reagent/medicine/morphine,
 		/datum/reagent/medicine/perfluorodecalin,
 		/datum/reagent/medicine/bicaridine,
 		/datum/reagent/medicine/kelotane,
 		/datum/reagent/medicine/antitoxin
 	)
```

Comment:
How comes there are 2 different things here if they all go into chem bags anyway. Personally I think if we want to add more of a specific reagent then roundstart_chems should instead be an assoc list with the value being the reagent amount

```
	var/list/roundstart_chems = list(
		/datum/reagent/medicine/epinephrine = 80,
		/datum/reagent/medicine/morphine = 120,
		/datum/reagent/medicine/perfluorodecalin = 80,
		/datum/reagent/medicine/bicaridine = 80,
		/datum/reagent/medicine/kelotane = 80,
		/datum/reagent/medicine/antitoxin = 80
	)
```